app/src/androidTest/java/org/sagebionetworks/research/mpower/TrackingTaskViewModelTest.java

```diff
@@ -99,8 +99,8 @@ protected SimpleTrackingItemConfig instantiateConfigFromSelection(@NonNull final
     @UiThreadTest
     public void initializeViewModel() {
         this.viewModel = new TestViewModel(TRACKING_STEP_VIEW, null);
-        this.viewModel.setTaskStartDate(START_DATE);
-        this.viewModel.setTaskEndDate(END_DATE);
+//        this.viewModel.setTaskStartDate(START_DATE);
```

Comment:
This wasn't compiling?

Reply:
Yea :\ @Robert-Kolmos is looking into it